kafkaconnect/reflect.go

```diff
@@ -129,6 +129,7 @@ func reflectInternal(v reflect.Value, t reflect.Type) *Schema {
 		for _, kv := range v.MapKeys() {
 			keyValue = kv
 			valValue = v.MapIndex(kv)
+			break
```

Comment:
Intended change?

Reply:
Yes, we just need a value from the map, so we don't need to range over all keys, the first value should work just fine.